doc/dev/workflow.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+This document describes the general developement workflow for
+developing and working with mixer in the broader context of Istio.
+This document includes information on building mixer properly and
+then building the associated containers in a local environment
+prior to prow updating the istio/istio dependency repositories.
+
+## Prepare tree
+
+Prepare the mixer repository and Go environment variables as
+specified in the 
+[contribution guidelines](https://github.com/istio/istio/blob/master/CONTRIBUTING.md).
+
+## Prepare a registry account on gcr.io
+
+Follow the
+[Google Container Registry Quickstart](https://https://cloud.google.com/container-registry/docs/quickstart)
+
+## Build and clean operations
```

Comment:
Can this just be a pointer to https://github.com/istio/istio/blob/master/devel/README.md so as to avoid repetition and divergence?

Reply:
fixed - please have a look at the new version.